config/routes.rb

```diff
@@ -15,5 +15,6 @@
     get "/healthcheck", to: "healthcheck#check"
 
     post "/unsubscribe/:id", to: "unsubscribe#unsubscribe"
+    post "/unsubscribe-all/:id", to: "unsubscribe#unsubscribe_all"
```

Comment:
Not sure it matters now since we've already got one in production, but I think a better route structure for both of these would be:

```ruby
delete "/subscriptions/:id", to: "unsubscribe#unsubscribe"
delete "/subscriber/:id", to: "unsubscribe#unsubscribe_all"
```

Reply:
That does look nicer - I can make this one a `DELETE` on subscriber, but would that be OK in terms of diverging?

Reply:
Haha OK

Reply:
Done